demos/common/cpp/utils/include/utils/common.hpp

```diff
@@ -39,6 +39,11 @@ constexpr std::size_t arraySize(const T (&)[N]) noexcept {
     return N;
 }
 
+template <typename T>
+T clamp(T value, T low, T high) {
+    return value<low ? low : (value>high ? high : value);
```

Comment:
```suggestion
    return value < low ? low : (value > high ? high : value);
```
Why is function called `clamp`? I think `clip` is more suitable here, similar function in python called the same.
And I would rename `low` and `high` to `minVal` and `maxVal` (or just `min` and `max`) respectively.

Reply:
There's std::clamp in C++17, as soon as it will be fully supported by our code, we'll switch to it. Signature is taken from that standard.